src/main/java/org/jabref/gui/groups/GroupTreeView.java

```diff
@@ -269,6 +269,10 @@ private void initialize() {
                 if (event.getButton() == MouseButton.SECONDARY) {
                     // Prevent right-click to select group
                     event.consume();
+                } else if (event.getTarget() instanceof StackPane pane) {
+                    if (pane.getStyleClass().toString().equals("arrow") || pane.getStyleClass().toString().equals("tree-disclosure-node")) {
```

Comment:
Maybe it's better to check for contains, because there could also be multiple style classes assigned.

Reply:
Thanks @calixtus , it has been updated with contains().
Really appreciate your review!